hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -849,7 +849,7 @@ func getAMI(nodePool *hyperv1.NodePool, region string, releaseImage *releaseinfo
 func ignConfig(encodedCACert, encodedToken, endpoint string) ignitionapi.Config {
 	return ignitionapi.Config{
 		Ignition: ignitionapi.Ignition{
-			Version: "3.1.0",
+			Version: "3.2.0",
```

Comment:
curious any particular motivation for the bump or just being at latest?

Reply:
In 4.9 and above we are using 3.2.0 -> https://github.com/openshift/installer/blob/1c538b8949f3a0e5b993e1ae33b9cd799806fa93/pkg/asset/ignition/bootstrap/common.go#L21